runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -98,6 +98,91 @@ function M.on_refresh(err, _, ctx, _)
   return vim.NIL
 end
 
+--- @class InlayHintOptions
+--- @field bufnr number|nil
+--- @field filter function|nil
+--- @field id number|nil
+--- @field name string|nil
+--- @field range { start: number[], end: number[] }|nil
+
+--- Returns inlay hints in the (optionally) given buffer, client, and range.
+--- This method does not make an LSP request, but just queries locally stored inlay hints.
+---
+--- @param options table|nil Optional table which holds the following optional fields:
+--- - bufnr (number|nil):
+---     Restrict hints to the clients attached to the given buffer, defaults to the current
+---     buffer (0).
+---
+--- - filter (function|nil):
+---     Predicate used to filter clients. Receives a client as argument and must return a
+---     boolean. Clients matching the predicate are included. Example: <pre>lua
+---                 -- Don't request inlay hints from typescript-language-server
+---                 vim.lsp.inlay_hint.get {
+---                   filter = function(client) return client.name ~= "tsserver" end
+---                 }
+---     </pre>
+---
+--- - id (number|nil):
+---     Restrict hints to the client with ID (client.id) matching this field.
+---
+--- - name (string|nil):
+---     Restrict hints to the client with name (client.name) matching this field.
+---
+--- - range (table|nil) Range to format.
```

Comment:
`to format` is probably a copy paste error?

Reply:
fixed :+1: